mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/MimePart.java

```diff
@@ -214,11 +213,6 @@ public HeaderCollection getHeaderCollection() {
         return headerCollection;
     }
 
-    @JsonProperty(JsonMessageConstants.HEADERS)
-    public Multimap<String, String> getHeaders() {
```

Comment:
How I will be able to search for mailing list for example? or for messages related to a message-id?

Reply:
You're being confused between MimePart header and email header.

Your ML or mimeMessage will be at the mail level, not at the mimepart level.